nbconvert/filters/markdown_mistune.py

```diff
@@ -104,15 +104,20 @@ def header(self, text, level, raw=None):
         html = super(IPythonRenderer, self).header(text, level, raw=raw)
         return add_anchor(html)
 
+    def escape_lt(self,text):
+        return text.replace('<','&lt;')
```

Comment:
Shouldn't we also be translating `>`? There's a function [html.escape()](https://docs.python.org/3/library/html.html#html.escape) which we might want to use.